src/lib/utils.ts

```diff
@@ -588,6 +589,16 @@ export function countNumberOfBytes(data: string) {
   return stripHexPrefix(data).length / 2;
 }
 
+export function countSignificantBits(data = '0x0') {
+  const bytes = hexToBytes(data);
+  for (let i = 0; i < bytes.length; i++) {
+    if (bytes[i] !== 0) {
+      return (bytes.length - i - 1) * 8 + 32 - Math.clz32(bytes[i]);
```

Comment:
What does this Math function do? It seems quite hard to understand this line of code.

Reply:
I am not sure to understand this line of code.

Reply:
It calculates the number of zero bits if the byte value had 32 bits. So the value for 0 is 32 and the value for 1 is 31 and for 2 is 30, 4 it's 29 and so on. The whole expression then returns the number of non zero bits.